consoles/sshVirtsh.pm

```diff
@@ -330,43 +330,98 @@ sub add_interface {
 sub add_disk {
     my ($self, $args) = @_;
 
-    my $file = $self->name . $args->{dev_id} . (($self->vmm_family eq 'vmware') ? ".vmdk" : ".img");
+    my $backingfile             = $args->{backingfile};
+    my $cdrom                   = $args->{cdrom};
+    my $name                    = $self->name;
+    my $file                    = $name . $args->{dev_id} . ($self->vmm_family eq 'vmware' ? '.vmdk' : '.img');
+    my $basedir                 = '/var/lib/libvirt/images/';
```

Comment:
I wonder if it would make sense to move this to a test/worker setting... But I see no reason to block from merging...

Reply:
@coolo does my comment make sense to you?

Reply:
to my personal taste this whole vmm_family thingy became unmaintainable long ago - there are so many specific commands in rather unexpected places. I wouldn't single out single commands. This should be rather refactored on a whole - but as long as @mnowaksuse is the only user of all of this there is no real need :(